packages/safe-tools-client/app/components/block-explorer-button/index.css

```diff
@@ -0,0 +1,36 @@
+.block-explorer-button-disabled {
```

Comment:
This looks like a modifier to the `block-explorer-button`. Use the modifier naming: https://getbem.com/naming/